serverless/lambda/src/test/java/uk/gov/di/lambdas/UpdateProfileHandlerTest.java

```diff
@@ -13,6 +13,7 @@
 import com.nimbusds.openid.connect.sdk.OIDCScopeValue;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import uk.gov.di.authentication.shared.services.ConfigurationService;
```

Comment:
does there need to be an integration test for this?

Reply:
good to have at least one

Reply:
Tests for the ConfigurationService or tests for the use of the ConfigurationService within UpdateProfileHandler ?

Reply:
update handler - as most handlers will have integration tests